lib/core.sh

```diff
@@ -396,13 +401,10 @@ function build_image_env(){
     sudo bash -c "echo 'export PATH=/opt/yaourt/bin:\$PATH' > ${maindir}/root/etc/profile.d/${CMD}.sh"
     sudo chmod +x ${maindir}/root/etc/profile.d/${CMD}.sh
 
-    info "Install ${NAME} script..."
-    sudo pacman --noconfirm --root ${maindir}/root -S git
-    _install_from_aur ${maindir} "${CMD}-git" "${CMD}.install"
-    sudo pacman --noconfirm --root ${maindir}/root -Rsn git
-
     info "Setting up the pacman keyring (this might take a while!)..."
-    sudo arch-chroot ${maindir}/root bash -c "pacman-key --init; pacman-key --populate archlinux"
+    sudo pacman --noconfirm --root ${maindir}/root -S psmisc
+    sudo ${maindir}/root/opt/junest/bin/jchroot ${maindir}/root bash -c "pacman-key --init; pacman-key --populate archlinux; killall gpg-agent"
```

Comment:
To my understanding, when a PPID is 1 it means its parent has completed, but the child has not and init process takes over..

I think all this hassle from dangling gpg-agent is because of some I/O streams misuse. I have to have a look